pkg/api/roles.go

```diff
@@ -195,29 +195,48 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Grants: []string{string(accesscontrol.RoleGrafanaAdmin)},
 	}
 
-	teamWriterGrants := []string{string(models.ROLE_ADMIN)}
-	if hs.Cfg.EditorsCanAdmin {
-		teamWriterGrants = append(teamWriterGrants, string(models.ROLE_EDITOR))
-	}
 	teamsWriterRole := accesscontrol.RoleRegistration{
 		Role: accesscontrol.RoleDTO{
 			Name:        "fixed:teams:writer",
 			DisplayName: "Team writer",
+			Description: "Read and manage team memberships.",
+			Group:       "Teams",
+			Version:     2,
+			Permissions: []accesscontrol.Permission{
```

Comment:
Resource manager creates default reader and writer roles when specified (see [here](https://github.com/grafana/grafana/pull/43951/files#diff-d76df9cc793ac30046ef234b57ff18bc567058fa4b567bece1ec33a1a1c9f7a3R76) as an example). So we could skip adding permissions here.

Additional note: This means we will have two fixed roles with different permissions. As currently resource manager does not allow to add extra permissions beside read and write.